currency/currency_test.go

```diff
@@ -159,3 +159,79 @@ func Test_FromMinorUnit_ZeroAmount(t *testing.T) {
 	assert.NoError(err)
 	assert.Zero(rs)
 }
+
+func Test_Format(t *testing.T) {
+	for n, tc := range map[string]struct {
+		amount       float64
+		currencyCode string
+		locale       string
+		want         string
+		wantErr      bool
+	}{
+		"Invalid currency": {
+			amount:       2048.172,
+			currencyCode: "SG",
+			locale:       "en",
+			want:         "",
+			wantErr:      true,
+		},
+		"Empty locale returns empty string": {
+			amount:       2048.179,
+			currencyCode: "USD",
+			locale:       "",
+		},
+		"Invalid locale returns in en locale": {
+			amount:       2048.179,
+			currencyCode: "USD",
+			locale:       "qwertyuiop",
+			want:         "$2,048.18",
+		},
```

Comment:
Wonder if empty locale should fallback to `en` locale like the invalid locale case. Thoughts?

Reply:
what is the default? I think in the library they return some default

Reply:
They return [empty struct based on their code](https://github.com/bojanz/currency/blob/4d4f7633e8bc01f21d170dd31e6a405055400e42/locale.go#L24).

https://go.dev/play/p/7fbAj85FnbK